include/grpcpp/ext/call_metric_recorder.h

```diff
@@ -23,10 +23,10 @@
 #include <grpcpp/support/slice.h>
 
 #include <memory>
+#include <optional>
```

Comment:
This include doesn't appear to actually be used anywhere, so it can probably be removed.

Reply:
Rm.